src/components/header/header.less

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/variables.less';
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+    height: 100px;
+    padding: 0 2rem;
+
+    h1 {
```

Comment:
Burde ikke style direkte på tags. Hva om man ønsker en annen h1 i headeren med en annen styling?

Reply:
ref https://frontstuff.io/you-need-to-stop-targeting-tags-in-css